Gemfile

```diff
@@ -38,8 +38,8 @@ gem 'momentjs-rails', '>= 2.9.0' # needed for human-friendly textual dates
 gem 'font-awesome-rails'
 
 # needed to run graders in the background
-gem 'delayed_job', github: 'dsander/delayed_job', branch: 'rails51' # FIXME: github
```

Comment:
I'm worried about deleting delayed_job entirely, this soon before the semester.  Is it possible to incorporate both for now, so we can migrate gently from one to the other?